util/metrics/import_metrics_files.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require "optparse"
+
+# This script is intended to be run on a puppet infrastructure node that had the puppet_metrics_collector
+# module installed and running.  Given a time window it will collect the json files generated by the
+# puppet_metrics_collector module and tar them into the tar file.  This includes jsons that are in tar
+# files already.  Those will be extracted and included if they are in the time window.
+
+# General namespace for metrics module
+module Metrics
+  # Main class for collecting metrics files into a tar file
+  #
+  # @author Randell Pelak
+  #
+  # @attr [integer] start_epoch Epoch seconds for the start of the collection window
+  # @attr [integer] end_epoch Epoch seconds for the end of the collection window
+  # @attr [string] metrics_dir The "output directory" of the metrics collector puppet module
+  # @attr [string] output_dir The directory to place the tarfile in.
+  # @attr [string] tar_file_name Name of the tar file to create with the metrics
+  # @attr [boolean] verbose Verbose output
+  # @attr [integer] poll_interval The "collection_frequency" of the metrics collector puppet module
+  # @attr [integer] archive_interval The frequency of the *_metrics_dir cron job for the metrics
+  #   collector module.  This is how often it tars up the json files.
+  #
+  class ImportMetricsFiles
```

Comment:
There are no spec tests for this class in this pr.

Reply:
Sorry; I meant to add a note that spec tests would be added once the implementation was approved. I wanted to avoid having to update them if there were significant updates to the implementation based on feedback.